### ISSUE TITLE
Is this project still active ?

### ISSUE THREAD

**Issue by motorsep** [code4game/glTFForUE4#8]
Lately I've had some issues with FBX static meshes (it seems that somehow UE4 interprets face normals on some triangles incorrectly when baking lighting) and I wanted to test glTF import. UE4's stock glTF support seems to be abandoned and so I am wondering if this project is still alive and is being actively supported/developed.

I am using 4.22.3 and going to update to 4.23 as soon as it's out.

**Comment by motorsep** [comment]
I am guessing it's dead :(

**Comment by alexchicn** [comment]
I am still in here. :)

**Comment by motorsep** [comment]
@alexchicn right, but is the project progressing? :) I checked the roadmap and it looked quite stale :(
